### PR TITLE
returning IPV6Address

### PR DESCRIPTION
**Pull request by chandra1-n**
JIRA: https://spotinst.atlassian.net/browse/PRD-2219 - Added returning IPV6Address support and examples for AWS Elastigroup and AWS ManagedInstance

### CODE REVIEW

File: service/elastigroup/providers/aws/aws.go [hunk 1]
```diff
@@ -874,6 +874,7 @@ type Instance struct {
 	PrivateIP        *string    `json:"privateIp,omitempty"`
 	PublicIP         *string    `json:"publicIp,omitempty"`
 	CreatedAt        *time.Time `json:"createdAt,omitempty"`
+	Ipv6Address      *string    `json:"ipv6Address,omitempty"`
```
Comment (Lironrad) [inline service/elastigroup/providers/aws/aws.go:877]: please rename to:
IPv6Address      *string

File: service/managedinstance/providers/aws/aws.go [hunk 2]
```diff
@@ -394,6 +394,7 @@ type StatusManagedInstanceOutput struct {
 	Status       *string    `json:"status,omitempty"`
 	CreatedAt    *time.Time `json:"createdAt,omitempty"`
 	LaunchedAt   *time.Time `json:"launchedAt,omitempty"`
+	Ipv6Address  *string    `json:"ipv6Address,omitempty"`
```
Comment (Lironrad) [inline service/managedinstance/providers/aws/aws.go:397]: please rename to:
IPv6Address *string

**Review by Lironrad — CHANGES_REQUESTED** [review]

File: service/elastigroup/providers/aws/aws.go (hunk 1 above)
Comment (chandra1-n) [reply service/elastigroup/providers/aws/aws.go]: renamed

File: service/managedinstance/providers/aws/aws.go (hunk 2 above)
Comment (chandra1-n) [reply service/managedinstance/providers/aws/aws.go]: renamed

**Review by Lironrad — APPROVED** [review]

**Pull request merged**